### ISSUE TITLE
update service is not working

### ISSUE THREAD

**Issue by manishrelani** [Dev-hwang/flutter_foreground_task#159]
In the latest version flutter_foreground_task: 6.0.0, The updateService Function is not updating the Same TaskHandler class, that I m providing in the callback function. it only updates the notification.

onStart function Of TaskHandler class calls only once when I m passing a new TaskHandler class.

This Functionality is working in the 5.2.1 version.

**Comment by Dev-hwang** [comment]
@manishrelani 

Due to difficulty in managing options and possible bugs, version 6.0.0 does not allow starting the same TaskHandler.

Is there a reason you want to start the same TaskHandler?

If you want to restart the service, it is recommended to use the function below:
```dart
FlutterForegroundTask.restartService();
```

**Comment by manishrelani** [comment]
Yes, I can use `FlutterForegroundTask.restartService();` to restart the service but the main reason was the same functionality with different notifications, because of that I was using `FlutterForegroundTask.updateService` with the same class. 

seems due to some reason it is intended to be like that.  I will close this issue.

also, It would be good to have the option to change the notification in `restartService`.

Thanks for your quick reply and this library.
